mu-plugins/central-hub/src/meta-data/helpers.php

```diff
@@ -12,6 +12,7 @@
 namespace spiralWebDB\Metadata;
 
 use KnowTheCode\ConfigStore as configStore;
+use function KnowTheCode\ConfigStore\str_starts_with;
```

Comment:
@rgadon107 This function is not used in the `helpers.php` file. Replace remove this import.

Reply:
>...remove this import.

Done.